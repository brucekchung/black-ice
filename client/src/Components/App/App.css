@import url('https://fonts.googleapis.com/css?family=Montserrat:300,400,700')

* {
  box-sizing: border-box;
}

.App {
  color: #eef0f2;
  background-color: #333;
  font-family: 'Montserrat', sans-serif;
  font-size: 1rem;
  min-height: 100vh;
}

.title {
  background-color: #222;
  font-size: 2.5rem;
  font-weight: 700;
  padding-top: 30px;
  text-align: center;
}

input,
button,
select {
  font-family: 'Montserrat', sans-serif;
  font-size: 1rem;
  font-weight: 300;
  height: 40px;
  outline: none;
  width: 250px;
}

input {
  padding: 0 10px;
}

input, select {
  background-color: #333;
  border: 1px solid #eef0f2;
  color: #eef0f2;
}

select {
  border-radius: 0px;
}

input::placeholder {
  color: #eef0f2;
  opacity: 0.9;
}

input:focus {
  border: 2px solid #0892a5;
}

button {
  color: #eef0f2;
  background-color: #0892a5;
  border: 1px solid #0892a5;
  font-weight: 400;
}

button:hover {
  background-color: #09A0B5;
  border: 1px solid #09A0B5;
  opacity: 1;
}