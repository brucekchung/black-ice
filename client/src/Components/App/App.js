import React, { Component } from 'react'
import './reset.css'
import './App.css'
import Main from '../Main/Main'
import Nav from '../Nav/Nav'

class App extends Component {
  render() {
    return (
      <div className="App">
        <Nav />
        <Main />
      </div>
    )
  }
}

export default App
