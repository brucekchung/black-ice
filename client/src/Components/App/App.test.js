import React from 'react'
import App from './App'
import { shallow } from 'enzyme'

describe('App', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<App />)
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
