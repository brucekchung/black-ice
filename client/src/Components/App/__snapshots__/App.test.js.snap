// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header
          className="App-header"
>
          <h1
                    className="App-title"
          >
                    Black Ice
          </h1>
</header>,
        <Main />,
      ],
      "className": "App",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h1
            className="App-title"
>
            Black Ice
</h1>,
          "className": "App-header",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Black Ice",
            "className": "App-title",
          },
          "ref": null,
          "rendered": "Black Ice",
          "type": "h1",
        },
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header
            className="App-header"
>
            <h1
                        className="App-title"
            >
                        Black Ice
            </h1>
</header>,
          <Main />,
        ],
        "className": "App",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1
              className="App-title"
>
              Black Ice
</h1>,
            "className": "App-header",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Black Ice",
              "className": "App-title",
            },
            "ref": null,
            "rendered": "Black Ice",
            "type": "h1",
          },
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
